src/frontend/packages/cloud-foundry/src/cf-entity-generator.ts

```diff
@@ -964,8 +965,12 @@ function generateCfOrgEntity(endpointDefinition: StratosEndpointExtensionDefinit
           status: getOrgStatus(org),
           name: org.entity.name,
           cfGuid: org.entity.cfGuid,
+          createdAt: moment(org.metadata.created_at).format('LLL'), // TODO: RC
         }),
         getLink: metadata => `/cloud-foundry/${metadata.cfGuid}/organizations/${metadata.guid}`,
+        getLines: () => ([
+          ['Creation  Date', (meta) => meta.createdAt]
```

Comment:
Should this be a double space? I removed one above on line 935, but not sure now?

Reply:
Visually they looked the same but have updated all references. We'll need to look at this process again when we do internationalisation.